internal/testdata/metric.go

```diff
@@ -218,19 +218,25 @@ func initExponentialHistogramMetric(hm pmetric.Metric) {
 	hdp0.SetZeroCount(1)
 	hdp0.SetScale(1)
 
-	// positive index 1 and 2 are values sqrt(2), 2 at scale 1
+	// at scale 1,
+	// postitive bucket index 0 is (1, sqrt(2)] (not represented)
+	// postitive bucket index 1 is (sqrt(2), 2]
+	// postitive bucket index 2 is (2, 2*sqrt(2)]
 	hdp0.Positive().SetOffset(1)
 	hdp0.Positive().BucketCounts().FromRaw([]uint64{1, 1})
-	// negative index -1 and 0 are values -1/sqrt(2), -1 at scale 1
+
+	// at scale 1
+	// negative bucket index 0 is -(1, sqrt(2)]
+	// negative bucket index -1 is -(sqrt(2)/2, 1]
 	hdp0.Negative().SetOffset(-1)
 	hdp0.Negative().BucketCounts().FromRaw([]uint64{1, 1})
 
 	// The above will print:
-	// Bucket (-1.414214, -1.000000], Count: 1
-	// Bucket (-1.000000, -0.707107], Count: 1
+	// Bucket [-1.414214, -1.000000), Count: 1
```

Comment:
Should these match the changes in [metrics_with_all_types.out](https://github.com/open-telemetry/opentelemetry-collector/pull/5938/files#diff-fcf207003d9dec15c485d510f4526edfb00f5c915c7251188e73eb7927e6f14a)?

Reply:
Yes.  Sorry, I at first changed the code to use %f like the explicit bucket histogram does, but after reading the tests for a while I decided it was worse to use %f and changed back to %.6f. Will update.